dashboard/app/models/pd/workshop.rb

```diff
@@ -192,7 +192,8 @@ def self.in_state(state)
       when STATE_ENDED
         where.not(started_at: nil).where.not(ended_at: nil)
       else
-        raise "Unrecognized state: #{state}"
+        raise "Unrecognized state: #{state}" if error_on_bad_state
+        none
```

Comment:
If all workshops should be in a good state I'd suggest logging something that should be picked up by HoneyBadger. It might be enough to just log an ERROR